### DISCUSSION TITLE
Mesh gen shrinkwrap normals

### DISCUSSION THREAD

**Discussion by TruffleFluffle** [chsh2/nijiGPen#47] [Ideas]
On one of joey carlino's videos he makes a spore creature creator. An interesting detail is he uses some shrinkwrap normals geo nodes to make it look like its remeshed without actually being remeshed. I took the same shrinkwrap normals geo nodes and tested it on nijigpens mesh gen, and it works perfectly. But it is time consuming to apply the geo nodes to each object if it is stopmotion. Perhaps you can integrate the shrinkwrap normals into the mesh gen so there can be a fake remesh. You could pick the object to shrinkwrap to the new mesh will look like its connected. 
![image](https://github.com/user-attachments/assets/b878e6c2-c642-479a-a533-343906175560)
![image](https://github.com/user-attachments/assets/4c55f5cb-dab8-4602-8c9e-a6851341d5b9)
![image](https://github.com/user-attachments/assets/8e563644-1ab2-45b7-ba2b-42874627b649)

**Comment by chsh2** [comment]
OK I will take a look at this shrinkwrap node group. And there are some general methods to speed up applying geometry nodes:

- Although the stopmotion mode generates a separate object for each frame, they can actually be joined together (`CTRL + J`), so that geometry node only needs to be set once.
- Blender has an operator `[Object]-[Link/Transfer Data]-[Copy Modifiers]` to copy modifiers from one object to all selected objects. Geometry nodes can also be copied in this way.
